docs/servers.rst

```diff
@@ -0,0 +1,57 @@
+.. _websocket-servers:
+
+Servers
+=======
+
+.. currentmodule:: trio_websocket
+
+Creating A Server
+-----------------
+
+This page goes into the details of creating a WebSocket server. Let's start by
+revisiting the example from the :ref:`server-tutorial`.
+
+.. code-block:: python
+    :linenos:
+
+    import trio
+    from trio_websocket import serve_websocket, ConnectionClosed
+
+    async def echo_server(request):
+        ws = await request.accept()
```

Comment:
API:  I'm wondering again about a convenience message iterator on the connection:
```python
try:
    ws = request.accept()
    async for message in ws:
        await ws.send_message(message)
except ConnectionClosed:
    break
```
or going further and encapsulating accept() as well via iterator on request:
```python
try:
    async for ws, message in request:
        await ws.send_message(message)
except ConnectionClosed:
    break
```
These receive-driven patterns preclude calling ping/pong/aclose, but I expect use of those to be relatively rare.

Use of these convenience iterators would be optional, so it's a question of value.

Reply:
I think it would probably be helpful, and it seems like pretty low effort. I've already opened #29 for this.